src/utils/FormInputHandling.ts

```diff
@@ -17,14 +17,65 @@ async function getData(currentPage: number, limit: number) {
 
 // post data to the server
 async function postFormData(formData: RowType) {
-  const currentDate = new Date().toLocaleDateString();
+  const { student, test, timeline, dateCreated } = formData;
+  let start_time = timeline.startDate + "T" + timeline.startTime + ":00";
+  let end_time = timeline.endDate + "T" + timeline.endTime + ":00";
+  const requestBody = {
+    name: test.name,
+    platform: test.platform,
+    platform_link: test.platformLink,
```

Comment:
this should be empty .. we don't have platform link yet

Reply:
updated

Reply:
updated where? i still see test.platformlink